packages/desktop-client/src/components/reports/ReportCard.tsx

```diff
@@ -119,10 +120,27 @@ type LayoutProps = {
 
 function Layout({ children, isEditing, menuItems, onMenuSelect }: LayoutProps) {
   const triggerRef = useRef(null);
-  const [menuOpen, setMenuOpen] = useState(false);
+  const viewRef = useRef(null);
+  const [menuOpen, setMenuOpen] = useState<null | 'context' | 'button'>(null);
+  const [crossOffset, setCrossOffset] = useState(0);
+  const [offset, setOffset] = useState(0);
+  const reportBudgetEnabled = useFeatureFlag('reportBudget');
```

Comment:
I don't think report budget is needed here, did you mean the dashboards feature flag?